tests/data/library_of_congress/bd20x24x.xml

```diff
@@ -262,9 +262,6 @@
     </datafield>
   </record>
   <record>
-    <datafield tag="240" ind1="1" ind2="0"/>
```

Comment:
Indicators convey values related to the subfields they contain. E.g. in this case:
https://www.loc.gov/marc/bibliographic/bd240.html

ind1 set to 1 means that the provided title is "Printed or displayed"... but... what title?